plugins/scaffolder/src/components/MultistepJsonForm/MultistepJsonForm.tsx

```diff
@@ -49,6 +49,61 @@ type Props = {
   fields?: FormProps<any>['fields'];
 };
 
+export function getUiSchemasFromSteps(steps: Step[]): UiSchema[] {
+  const uiSchemas: Array<UiSchema> = [];
+  steps.forEach(step => {
+    if (!step.schema || !step.schema.properties) return [];
+
+    const schemaProps = step.schema.properties as JsonObject;
+    for (const key in schemaProps) {
+      if (Object.prototype.hasOwnProperty.call(schemaProps, key)) {
```

Comment:
Why not `schemaProps.hasOwnProperty`?

Reply:
It is here just because de autocomplete... :/ Fixing...